SeleniumPracticalExercise/PageObjects/AddingEmployee.cs

```diff
@@ -0,0 +1,27 @@
+﻿using OpenQA.Selenium;
+using SeleniumPracticalExercise.PageObjects.Common;
+
+namespace SeleniumPracticalExercise.PageObjects
+{
+    public class AddingEmployee : BasePageLocal
```

Comment:
I changed it to AddingEmployee. Probably wrong name now thinking about it. I probably should have done. AddName. But I thought thats what it was AddingEmployee since that was the page.?

Reply:
Renamed it to AddEmployeePage